tests/contracts/Testcontracts.ml

```diff
@@ -155,11 +155,14 @@ let add_tests env =
     let mappairtests = "mappair" >:::(build_contract_tests env "mappair" succ_code 1 5) in
     let bookstoretests = "bookstore" >:::(build_contract_tests env "bookstore" succ_code 1 10) in
     let mappairtests_f = "mappair" >:::(build_contract_tests env "mappair" fail_code 6 8) in
+    let nonfungibletokentests = "nonfungible-token" >:::(build_contract_tests env "nonfungible-token" succ_code 1 12) in
+    let nonfungibletokentests_expected_f = "nonfungible-token" >:::(build_contract_tests env "nonfungible-token" succ_code 21 27) in
```

Comment:
am I correct that this gap in numbering is intentional? (for future success tests)?

Reply:
Yes it is intentional.